ui/lib/replication/addon/templates/mode.hbs

```diff
@@ -1,57 +1,62 @@
 <section class="section">
   <div class="container is-widescreen">
-    {{#if model.replicationAttrs.replicationEnabled}}
-      <PageHeader as |p|>
-        <p.top>
-          <nav class="key-value-header breadcrumb">
+    {{#if (and (eq model.drMode "secondary") (eq model.drModeInit "primary"))}}
```

Comment:
This is the only change to this file, the rest of just indenting in to accommodate the new conditional